### PR TITLE
Issue #7: Added Community Hardware Connectors

### PR DESCRIPTION
**Pull request by alexdgilbert**
Added Community Hardware Connectors to Community repository

### CODE REVIEW

File: src/main/connector/hardware/MIB2Solaris/MIB2Solaris.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+extends:
+- ../Hardware/Hardware
+- ../MIB2-header/MIB2-header
+- ../SunEthernetDriver-header/SunEthernetDriver-header
```
Comment (NassimBtk) [inline src/main/connector/hardware/MIB2Solaris/MIB2Solaris.yaml:5]: `../SunEthernetDriver-header/SunEthernetDriver-header` is not present.

Comment (bertysentry) [reply src/main/connector/hardware/MIB2Solaris/MIB2Solaris.yaml:5]: MIB2Solaris should be removed (moved back to Enterprise)

**Review by bertysentry — COMMENTED** [review]
Should we improve the style of this code? Ideas:

* Rename embedded files to something meaningful (at least something ending with `.awk`)
* Inline translation tables
* Remove unnecessary quotes and double quotes
* Use prettier (or something like that) on the AWK scripts (some of them are really ugly)

**Comment by NassimBtk** [comment]
> Should we improve the style of this code? Ideas:
> 
> * Rename embedded files to something meaningful (at least something ending with `.awk`)
> * Inline translation tables
> * Remove unnecessary quotes and double quotes
> * Use prettier (or something like that) on the AWK scripts (some of them are really ugly)

I am fine with all the changes except the `Inline translation tables` because it isn't supported yet by the hdf-maven-plugin.

**Comment by alexdgilbert** [comment]
Case HC-1628 has been created internally to loop in QA.

**Review by bertysentry — DISMISSED** [review]
Nice!

**Review by NassimBtk — APPROVED** [review]

**Pull request merged**